app/processors/external_app_records_processor.rb

```diff
@@ -3,13 +3,21 @@
 class ExternalAppRecordsProcessor < ApplicationProcessor
   include Sortable
   SORTABLE_FIELDS = %w[
-    server_id servers.name servers.numero external_name external_id external_serial frames.name
+    server_id server.name server.numero external_name external_id external_serial frames.name
   ].freeze
 
   map :frame_ids, filter_with: :non_empty_array do |frame_ids:|
     raw.joins(:frame).where(frame: { id: frame_ids })
   end
 
+  map :modele_ids, filter_with: :non_empty_array do |modele_ids:|
+    raw.joins(:server).where(servers: { modele_id: modele_ids })
+  end
+
+  map :server_name do |server_name:|
+    raw.joins(:server).where(Server.arel_table[:name].matches("%#{server_name}%"))
```

Comment:
```suggestion
    raw.joins(:server).where(Server.arel_table[:name].alias(:server).matches("%#{server_name}%"))
```

Try this to fix the issue

Reply:
It worked, the alias must be set before the `[:name]`.  However, I had to modify the Modele filter to use `server` in the query.  
```ruby
map :modele_ids, filter_with: :non_empty_array do |modele_ids:|
  raw.joins(:server).where(server: { modele_id: modele_ids })
end
```

Reply:
Now we have the same issue but on the tests (spec/processors/external_app_records_processor_spec.rb)... I've been working on a fix, but I'm not sure it is the right way of doin it